aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -294,6 +294,16 @@ builder.WebHost.ConfigureKestrel(serverOptions =>
 
 For more information about this feature and how to use .NET and gRPC to create an IPC server and client, see <xref:grpc/interprocess>.
 
+### Performance improvements to named pipes transport
+
+We’ve improved named pipe connection performance. Kestrel’s named pipe transport now accepts connections in parallel, and reuses <xref:System.IO.Pipes.NamedPipeServerStream> instances.
+
+Time to create 100,000 connections:
+
+* Before : 5.916 seconds
+* After     : 2.374 seconds
```

Comment:
@guardrex  remind me how to get an extra space here so the numbers line up.

Reply:
Did you try `&nbsp;`

Reply:
> Did you try `&nbsp;`

yup, that works.